From eeba76789553966c9d58389b77cfd62bb81466c6 Mon Sep 17 00:00:00 2001
From: Riku <riksu9000@gmail.com>
Date: Sat, 6 Jul 2024 19:29:31 +0300
Subject: [PATCH] 70-mouse.hwdb: Added Glorious Model O DPI

This rule might also apply to the regular Model O (no minus), which has the same specifications.

(cherry picked from commit cd8eb9605f1a1f2fe1180382201d23548cad7a1c)

Related: RHEL-5950
---
 hwdb.d/70-mouse.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 0379eecf07..20079d6350 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -248,6 +248,14 @@ mouse:usb:v1ea7p000b:name:2.4G RF Mouse:*
 mouse:usb:v04d9p0499:name:*:*
  MOUSE_DPI=800@125
 
+##########################################
+# Glorious
+##########################################
+
+# Glorious Model O Minus
+mouse:usb:v258ap0036:name:Glorious Model O:*
+ MOUSE_DPI=400@1000 800@1000 *1600@1000 3200@1000
+
 ##########################################
 # HandShoe Mouse
 ##########################################
